docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -34,6 +34,7 @@
 |spring.cloud.loadbalancer.health-check.initial-delay |  | Initial delay value for the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.interval | 25s | Interval for rerunning the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.path |  | 
+|spring.cloud.loadbalancer.hint |  | Allows setting the value of <code>hint</code> that is passed on to the LoadBalancer request and can subsequently be used in {@link ReactiveLoadBalancer} implementations.
```

Comment:
LoadBalancer and ReactiveLoadBalancer should be marked as code. You can use backticks to do it if you want something shorter than the code element. So `LoadBalancer` rather than <code>LoadBalancer</code>.

Reply:
This is file is automatically generated.